api/v1alpha1/combination_types.go

```diff
@@ -74,6 +54,8 @@ type Argument struct {
 type CombinationStatus struct {
 	// Conditions represents the current condition of the Combination.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Represents the evaluation to this combination once processed
+	Evaluations []string `json:"evaluations,omitempty"`
```

Comment:
Note: later let's discuss if there's a potential API improvement that will make it easier to identify which unique key/value set generated a particular element of the `Evaluations` set; e.g.

```go
// ...
Evaluations []Evaluation
// ...
type Evaluation struct {
    ArgCombo map[string]string // key -> value
    Result string
}
```